styles/main.css

```diff
@@ -0,0 +1,129 @@
+* {
+	box-sizing: border-box;
+	padding: 0;
+	margin: 0;
+}
+
+html {
+	scroll-behavior: smooth;
+}
+
+.grid-container {
+	display: grid;
+	gap: 0;
+	grid-template-areas:
+		'h m m m'
+		'n m m m'
+		'n m m m'
+		'n f f f';
+}
+
+header {
+	grid-area: h;
+	height: 10vh;
+	width: 20vw;
+	padding-top: 4%;
```

Comment:
Love the vw and vh usage! For padding, I would recommend using `rem` or `em` instead of `%` for precision.